contents/docs/self-host/deploy/snippets/upgrading.mdx

```diff
@@ -15,19 +15,51 @@ helm search repo posthog
 ```
 Compare the numbers for the chart version (in the format `posthog-{major}.{minor}.{patch}` - for example, `posthog-3.15.1`) when running the commands above. If the upgrade is for a major version, check the [upgrade notes](/docs/self-host/deploy/upgrade-notes) before moving forward.
 
-3. Run the upgrade
+3. (optional) Grab the up-to-date values.yaml file and save to values.yaml
```

Comment:
There is a better way: `helm upgrade --use-values` & you can set whatever you want extra or just overrides from cli or values.yaml they would now append to whatever is live